packages/bpk-component-dialog/src/BpkDialog.scss

```diff
@@ -29,6 +29,12 @@
     }
   }
 
+  &--with-flare {
+    height: $bpk-spacing-base * 10;
+    border-radius: inherit;
```

Comment:
This is so that the corners will round to the same radius as the modal container itself

Reply:
Let's add a quick comment to that effect in the code, then it's documented for the future too